src/main/java/org/openpnp/gui/JobPanel.java

```diff
@@ -1254,6 +1257,14 @@ public void actionPerformed(ActionEvent arg0) {
                         MainFrame.get().getCameraViews().ensureCameraVisible(camera);
                         Location location = getSelection().getLocation();
                         MovableUtils.moveToLocationAtSafeZ(camera, location);
+                        try {
+                            Map<String, Object> globals = new HashMap<>();
+                            globals.put("camera", this);
```

Comment:
This should probably be `globals.put("camera", camera);` and not `globals.put("camera", this);`, otherwise the scripts are going to get the instance of the action listener instead of the camera. Same with the other instances of this call in the PR.